resources/js/curl-to-go.js

```diff
@@ -196,21 +328,16 @@ function curlToGo(curl) {
 			relevant.url = cmd._[1]; // position 1 because index 0 is the curl command itself
 
 		// gather the headers together
-		if (cmd.H)
-			relevant.headers = relevant.headers.concat(cmd.H);
 		if (cmd.header)
 			relevant.headers = relevant.headers.concat(cmd.header);
 		relevant.headers = parseHeaders(relevant.headers)
 
 		// set method to HEAD?
-		if (cmd.I || cmd.head)
+		if (cmd.head)
 			relevant.method = "HEAD";
 
-		// between -X and --request, prefer the long form I guess
```

Comment:
In reality, curl doesn't distinguish between `-X` and `--request`, it takes the last one, regardless of whether it uses the short or long form, so this PR makes curl-to-Go act more like curl.